config/urls.py

```diff
@@ -112,6 +112,10 @@
         "inclusion-connect-get-logout-info/",
         dora.core.views.inclusion_connect_get_logout_info,
     ),
+    path(
+        "inclusion-connect-update-info/",
```

Comment:
```suggestion
        "inclusion-connect-get-update-info/",
```
pour être consistent avec les deux méthodes précédentes.